srcpkgs/hyperrogue/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'hyperrogue'
+pkgname=hyperrogue
+version=11.3f
+revision=1
+archs="x86_64*"
```

Comment:
this shouldn't be set unless you are 100% sure this only works exclusively on that arch, which I don't think is the case